pandas/core/panel.py

```diff
@@ -197,7 +199,7 @@ def _init_dict(self, data, axes, dtype=None):
             haxis = Index(ks)
 
         for k, v in compat.iteritems(data):
-            if isinstance(v, dict):
+            if isinstance(v, MutableMapping):
```

Comment:
`MutableMapping` is a subclass of `Mapping`. 
But here if we pass a `Mapping` that's not mutable, it'll raise. 
We could have a `is_mutable_dict_like`? Although `isinstance` is clear here (and generally things seem to be going towards `isinstance`?)